standalone/src/main/java/io/pravega/local/SingleNodeConfig.java

```diff
@@ -16,21 +16,26 @@
 
 public class SingleNodeConfig {
     //region config names
+    public final static String PROPERTY_FILE = "singlenode.configurationFile";
+
     public final static Property<Integer> ZK_PORT = Property.named("zkPort", 4000);
     public final static Property<Integer> SEGMENTSTORE_PORT = Property.named("segmentstorePort", 6000);
     public final static Property<Integer> CONTROLLER_PORT = Property.named("controllerPort", 9090);
     public final static Property<Integer> REST_SERVER_PORT = Property.named("restServerPort", 9091);
-    public final static Property<String> CERT_FILE = Property.named("certFile", "");
-    public final static Property<String> KEY_FILE = Property.named("keyFile", "");
-    public final static Property<String> PASSWD_FILE = Property.named("passwdFile", "");
+    public final static Property<String> CERT_FILE = Property.named("certFile", "../config/cert.pem");
```

Comment:
I'd much rather you left these as they were (empty) and only modify the config file.

Reply:
@andreipaduroiu - The rationale for setting defaults for the certificate, key, keystore and truststore files here was so that it's easier to deploy TLS-enabled Pravega standalone mode. With this change, one can do so simply by setting "singlenode.enableTls=true" if one is OK with using provided default certificates, keys, keystores and truststores. Without these defaults, the user will have to mandatorily specify all of the following properties for TLS, as well: 
* singlenode.certFile
* singlenode.keyFile
* singlenode.keyStoreJKS
* singlenode.keyStoreJKSPasswordFile
* singlenode.trustStoreJKS

What do you say?

Reply:
Seems to me that the standalone mode has a config file with good defaults.   Putting the defaults into code may inadvertently affect the embedded mode and would be duplicative?

It is probably a bad practice to ship any private keys or default passwords, since you wouldn't want anyone to use them.  

In any case, please ensure that these changes don't affect the embedded mode.